.github/workflows/ci.yaml

```diff
@@ -84,6 +84,13 @@ jobs:
   enforcer:
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: zulu
+          java-version: 11
+      - run: java -version
+      - run: mvn enforcer:enforce@enforce -T 1C
```

Comment:
I don't think you intend to make change here

Reply:
oof thank you. This was unintentional.